car_app.py

```diff
@@ -1,3 +1,12 @@
 class Car:
-    pass
+    def __init__(self, name, color, wheels):
+    	self.name = name
+		self.wheeles = wheeles
```

Comment:
Aici o sa crape caci in argument vine la `wheels` iar la definirea variabilei clasei e `wheeles`